### ISSUE TITLE
No able to minimize presentation

### ISSUE THREAD

**Issue by Braehler** [bigbluebutton/bigbluebutton#14217]
**Describe the bug**
Not able to minimize the presentation.  

**To Reproduce**
1. Start a meeting of your choise'...'
2. Click upload presentation or use the default one
3. try to minimize the presentation
4. Only the option to have it on fullscreen

**Expected behavior**
A button, like in 2.3 where the user can close or minimize the presentation

**Actual behavior**
Just the option to make it full width

**Screenshots**

**BBB version:**
BigBlueButton continually evolves. Providing the version/build helps us to pinpoint when an issue was introduced.
Example:
$ sudo bbb-conf --check | grep BigBlueButton
BigBlueButton Server 2.4.0

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome and FF
 - Version: both latest stable versions

**Smartphone (please complete the following information):**
 - Browser [e.g. stock browser, Safari]

**Additional context**
![minimize](https://user-images.githubusercontent.com/29776943/151330522-0097862c-5702-4c33-9a41-e06a7889b699.PNG)

**Comment by hiroshisuga** [comment]
on the right bottom you have the minimise/reopen the presentation button. It's true that it's not intuitive.

**Comment by Braehler** [comment]
@hiroshisuga 
alright, thanks a lot, Just like sou said, thats realy not intiutive. But I´m going to show that to our teachers and students.
One more thing: As far as I remember, it was possible to rezize the presentation as well. Is this still possible?

**Comment by hiroshisuga** [comment]
Yes, everything related to the slide are now located on the narrow gray area just under the slide. You can also zoom in/out by mouse wheel. Don't ask me why still the minimise slide button is at the very bottom;-)

**Comment by antobinary** [comment]
Please give this a try:
https://github.com/bigbluebutton/bigbluebutton/blob/v2.4.x-release/bigbluebutton-html5/private/config/settings.yml#L601
Set `oldMinimizeButton: true` , restart BBB and tell us if this achieved the desired result.
